spec/requests/charges_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
```

Comment:
Fine, but you should test all your actions! Not only create and new! Also you should test with invalid parameters